### PR TITLE
Update chapter11

### PR DESCRIPTION
**Pull request by danielvdende**

### CODE REVIEW

File: chapter11/compose.override.yaml [hunk 1]
```diff
@@ -49,5 +49,11 @@ services:
     # yamllint enable rule:line-length
     user: "0:0"
 
+  movie-reviews:
+    build: ./docker/movie-reviews-api
+    image: manning-airflow/events-api
```
Comment (krisgeus) [inline chapter11/compose.override.yaml:54]: Name is different from build context directory. I would align them. I believe events-api might be already use din other chapters so I suggest to use the reviews-api as name

File: chapter11/docker/movie-reviews-api/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.11-slim
```
Comment (krisgeus) [inline chapter11/docker/movie-reviews-api/Dockerfile:1]: Why not use python 12? we use it in other chapters I believe

File: chapter11/compose.override.yaml (hunk 1 above)
Comment (danielvdende) [reply chapter11/compose.override.yaml]: good catch, fixed.

**Review by krisgeus — APPROVED** [review]
LGTM

**Pull request merged**